### ISSUE TITLE
devDependencies defined in package scripts not detected.

### ISSUE THREAD

**Issue by robhaswell** [depcheck/depcheck#200]
Thank you for this wonderful tool. However, `depcheck` seems to think that my project is not using `standard`, yet it is used in my package scripts:

```json
{
  "scripts": {
    "start": "node app",
    "test": "npm run check-style && (istanbul cover --include-all-sources tape -- '!(node_modules)/**/*.test.js' | tap-dot) && npm run check-coverage && npm run check-security",
    "quick-test": "tape '!(node_modules)/**/*.test.js' | ./node_modules/.bin/tap-spec",
    "check-test": "istanbul cover --include-all-sources tape -- '!(node_modules)/**/*.test.js' | tap-dot",
    "check-security": "nsp check",
    "check-coverage": "istanbul check-coverage",
    "check-style": "standard",
    "lint": "standard"
  }
}
```

depcheck --json | json_pp:
```json
{
   "using" : {
      "jsonwebtoken" : [
         "/Users/cloudfind/Projects/jetski/lib/authenticate.js",
         "/Users/cloudfind/Projects/jetski/lib/test/authenticate.test.js"
      ],
      "nsp" : [
         "/Users/cloudfind/Projects/jetski/package.json"
      ],
      "supertest" : [
         "/Users/cloudfind/Projects/jetski/lib/test/app.test.js",
         "/Users/cloudfind/Projects/jetski/lib/test/authenticate.test.js"
      ],
      "cookies" : [
         "/Users/cloudfind/Projects/jetski/lib/authenticate.js"
      ],
      "bole" : [
         "/Users/cloudfind/Projects/jetski/app.js",
         "/Users/cloudfind/Projects/jetski/lib/logger.js"
      ],
      "helmet" : [
         "/Users/cloudfind/Projects/jetski/app.js",
         "/Users/cloudfind/Projects/jetski/web/server.js"
      ],
      "tape" : [
         "/Users/cloudfind/Projects/jetski/lib/test/app.test.js",
         "/Users/cloudfind/Projects/jetski/lib/test/append-user-info.test.js",
         "/Users/cloudfind/Projects/jetski/lib/test/authenticate.test.js",
         "/Users/cloudfind/Projects/jetski/lib/test/sqs-prototype.test.js"
      ],
      "response-time" : [
         "/Users/cloudfind/Projects/jetski/app.js",
         "/Users/cloudfind/Projects/jetski/web/server.js"
      ],
      "proxyquire" : [
         "/Users/cloudfind/Projects/jetski/lib/test/app.test.js"
      ],
      "morgan" : [
         "/Users/cloudfind/Projects/jetski/app.js",
         "/Users/cloudfind/Projects/jetski/web/server.js"
      ],
      "tap-dot" : [
         "/Users/cloudfind/Projects/jetski/package.json"
      ],
      "express" : [
         "/Users/cloudfind/Projects/jetski/app.js",
         "/Users/cloudfind/Projects/jetski/lib/test/authenticate.test.js",
         "/Users/cloudfind/Projects/jetski/web/server.js"
      ],
      "mississippi" : [
         "/Users/cloudfind/Projects/jetski/lib/sqs-prototype.js"
      ],
      "istanbul" : [
         "/Users/cloudfind/Projects/jetski/package.json"
      ],
      "aws-sdk" : [
         "/Users/cloudfind/Projects/jetski/lib/sqs.js"
      ],
      "hat" : [
         "/Users/cloudfind/Projects/jetski/app.js",
         "/Users/cloudfind/Projects/jetski/web/server.js"
      ]
   },
   "invalidFiles" : {},
   "dependencies" : [
      "body-parser",
      "cluster",
      "ejs",
      "fs",
      "metrics",
      "prettyjson"
   ],
   "missing" : {},
   "devDependencies" : [
      "garnish",
      "nodemon",
      "standard",
      "watchify"
   ],
   "invalidDirs" : {}
}
```

Many thanks,

**Comment by junedev** [comment]
I am experiencing the same a lot of times. Here is one example where nodemon is considered unused.
package.json:
```
"scripts": {
    "build": "tsc -p src/",
    "build:watch": "tsc -p src/ -w",
    "watch": "concurrently \"npm run build:watch\" \"nodemon app.js\"",
    "start": "node app.js",
    "lint": "tslint ./src/**/*.ts -t verbose"
}
```

depcheck --json:
```
{
    "dependencies": [
        "@angular/forms",
        "@angular/http",
        "@angular/router",
        "angular-in-memory-web-api",
        "systemjs",
        "core-js",
        "node-sass",
        "request",
        "socket.io",
        "twit"
    ],
    "devDependencies": [
        "@types/node",
        "nodemon"
    ],
    "missing": {},
    "using": {
        "morgan": [
            "/home/xyz/app.js"
        ],
        "express": [
            "/home/xyz/app.js"
        ],
        "node-sass-middleware": [
            "/home/xyz/app.js"
        ],
        "concurrently": [
            "/home/xyz/package.json"
        ],
        "tslint": [
            "/home/xyz/package.json"
        ],
        "typescript": [
            "/home/xyz/package.json"
        ],
        "@angular/core": [
            "/home/xyz/src/app/app.module.ts",
            "/home/xyz/src/app/app.component.ts",
            "/home/xyz/src/main.ts",
            "/home/xyz/public/app/app.component.js",
            "/home/xyz/public/app/app.module.js",
            "/home/xyz/public/main.js"
        ],
        "@angular/platform-browser": [
            "/home/xyz/src/app/app.module.ts",
            "/home/xyz/src/main.ts",
            "/home/xyz/public/app/app.module.js",
            "/home/xyz/public/main.js"
        ],
        "rxjs": [
            "/home/xyz/src/app/app.module.ts",
            "/home/xyz/src/app/app.component.ts",
            "/home/xyz/public/app/app.component.js",
            "/home/xyz/public/app/app.module.js"
        ],
        "zone.js": [
            "/home/xyz/src/app/app.module.ts",
            "/home/xyz/src/app/app.component.ts",
            "/home/xyz/public/app/app.component.js",
            "/home/xyz/public/app/app.module.js"
        ],
        "@angular/common": [
            "/home/xyz/src/app/app.module.ts",
            "/home/xyz/src/main.ts",
            "/home/xyz/public/app/app.module.js",
            "/home/xyz/public/main.js"
        ],
        "@angular/platform-browser-dynamic": [
            "/home/xyz/src/main.ts",
            "/home/xyz/public/main.js"
        ],
        "@angular/compiler": [
            "/home/xyz/src/main.ts",
            "/home/xyz/public/main.js"
        ]
    },
    "invalidFiles": {},
    "invalidDirs": {}
}
```

**Comment by VincentLanglet** [comment]
@rumpl This issue can be closed thanks to this PR https://github.com/depcheck/depcheck/pull/358

**Comment by rumpl** [comment]
Thank you for your work ^^
